tools/lint/src/main/java/com/grab/lint/ProjectXmlCreator.kt

```diff
@@ -4,8 +4,10 @@ import com.android.SdkConstants
 import java.io.File
 import java.nio.file.Path
 
+private fun List<String>.lintJars(): List<String> =
```

Comment:
Can we use sequence here since there are cascading map and filter operations? 

it.absolutePath will break cross machine caching, instead try with `Paths.get(it).resolve("lint.jar").exists()`. And use the Paths.get result instead of absolute path.